### DISCUSSION TITLE
Why Home Assistant can restore my network no problem but Z2M can't?

### DISCUSSION THREAD

**Discussion by CaliLuke** [Koenkk/zigbee2mqtt#14434] [General]
I had to rebuild my home assistant server after a catastrophic crash. Since I needed to get going faster I started by building a docker install first (no add-ons) so I used the built in zigbee extension and it found EVERYTHING I had paired with the coordinator, very impressive. Of course naming and rooms were gone but not having to rebuild the entire network was a huge time saver.

As I got a replacement for my hardware, I then proceeded to build a supervised version,  added the same coordinator and... nothing. Why doesn't zigbee2mqtt recognizes the zigbee devices that are paired with the coordinator?

Is there a way to "hint" the software by editing configuration files if necessary?

I have downloaded a config backup from the HA Zigbee extension, here attached.

[ZHA backup 2022-10-11T06-14-38.974Z.txt](https://github.com/Koenkk/zigbee2mqtt/files/9751880/ZHA.backup.2022-10-11T06-14-38.974Z.txt)

**Comment by ingoratsdorf** [comment]
Yup, I'd like to know the same thing.
There's a few topics on the net about migration from ZHA as well and .... no hints or answers.

**Reply by trankillity** [reply]
I'm having some significant issues with my SkyConnect and a few devices on ZHA, so am considering migrating to Z2M as well - but not having any luck finding best practices on moving over to Z2M :(

**Reply by ingoratsdorf** [reply]
I shared some comments here about a migration. The issue is the database, not the coordinator ;-)
https://community.home-assistant.io/t/zha-to-zigbee2mqttt-migration-using-backup-file/473687/11?u=ingoratsdorf

**Reply by Hedda** [reply]
Regarding migration from ZHA to Zigbee2MQTT see -> https://github.com/Koenkk/zigbee2mqtt/discussions/16739 (and also https://github.com/Koenkk/zigbee2mqtt/discussions/16478)
